youtube_dl/extractor/vvvvid.py

```diff
@@ -156,3 +163,56 @@ def f(m):
             'view_count': int_or_none(video_data.get('views')),
             'like_count': int_or_none(video_data.get('video_likes')),
         }
+
+
+class VVVVIDShowIE(VVVVIDIE):
+    _VALID_URL = r'https?://(?:www\.)?vvvvid\.it/(?:#!)?(?:show)/(?P<show_id>\d+)/(?P<id>[^/]+)(?:\?|\&|/$|$)'
+    _TESTS = [{
+        'url': 'https://www.vvvvid.it/show/156/psyco-pass',
+        'info_dict': {
+            'id': '156',
+            'title': 'Psycho-Pass',
+        },
+        'playlist_count': 46,
+    }]
+
+    def _real_extract(self, url):
+        show_id, video_id = re.match(self._VALID_URL, url).groups()
+
+        response = self._download_info(
+            'https://www.vvvvid.it/vvvvid/ondemand/%s/seasons/' % show_id, video_id)
+
+        show_infos = self._download_info(
+            'https://www.vvvvid.it/vvvvid/ondemand/%s/info/' % show_id, video_id)
+
+        show_title = show_infos['data'].get('title')
+
+        entries = []
+        lang = 'ITA'
+        for season in response['data']:
+            if 'giapponese' in season.get('name').lower():
+                lang = 'JP'
+            for episode in season.get('episodes'):
+                season_id = str_or_none(episode.get('season_id'))
+                video_id = str_or_none(episode.get('video_id'))
+                if not season_id or not video_id:
+                    continue
+
+                video_url = '/'.join([url, season_id, video_id])
+
+                # create video title as "ep. title [lang]"
+                episode_number = episode.get('number')
+                video_title = '%s. %s [%s]' % (
```

Comment:
no, the title should match what is used in `VVVVIDIE`, and also this would prevent users from customizing the output filename(ex: the user may prefer to use `ep - title`).